lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -58,7 +58,8 @@ def configure
         # rubocop:disable Metrics/MethodLength
         def call(env)
           # configure the Rack middleware once
-          configure()
+          configure
+          clean_context
```

Comment:
we can clean the context all the time; another PR makes this process safe even if the `@tracer` is not defined

Reply:
Reference: https://github.com/DataDog/dd-trace-rb/pull/209